tests/server/flow-event.js

```diff
@@ -705,6 +705,56 @@ define([
       }
     },
 
+    'call flowEvent with invalid loaded timing': {
+      beforeEach () {
+        flowMetricsValidateResult = true;
+        setup({
+          events: [
+            { offset: 31536000000, type: 'loaded' }
+          ],
+        }, 1000);
+      },
+
+      'process.stderr.write was called three times': () => {
+        assert.equal(process.stderr.write.callCount, 3);
+      },
+
+      'second call to process.stderr.write was correct': () => {
+        const arg = JSON.parse(process.stderr.write.args[0][0]);
+        assert.equal(arg.event, 'flow.performance.network');
+      },
+
+      'third call to process.stderr.write was correct': () => {
+        const arg = JSON.parse(process.stderr.write.args[1][0]);
+        assert.equal(arg.event, 'flow.performance.server');
+      },
+
+      'fourth call to process.stderr.write was correct': () => {
+        const arg = JSON.parse(process.stderr.write.args[2][0]);
+        assert.equal(arg.event, 'flow.performance.client');
+      }
+    },
+
+    'call flowEvent with invalid navigationTiming': {
+      beforeEach () {
+        flowMetricsValidateResult = true;
+        setup({
+          events: [
+            { offset: 1000, type: 'loaded' }
```

Comment:
is this event invalid because the offset is the same as `timeSinceFlowBegin`? I'm a bit confused why it's invalid.